### DISCUSSION TITLE
When I retime the drawings, it doesn’t show up. Why?

### DISCUSSION THREAD

**Discussion by Firefly65536** [keiwando/toonsquid#237] [Q&A] [answered]
![EE11DE83-D607-4A17-A66A-32D58D72C718](https://user-images.githubusercontent.com/121312948/229980333-c37fe368-710e-421c-8198-89e58632544c.png)
![2C6FFD1E-28E8-4564-A0E0-AF01F9550B2A](https://user-images.githubusercontent.com/121312948/229980342-29abde1f-1d59-40d1-be80-f7374eb14c2b.png)

**Comment by Cat-Butler** [comment, marked as answer]
The fact you have a Time channel seems to indicate this is a Symbol layer.
Symbols and scenes can sometimes interact in surprising ways.

Controlling the manifestation of the effect version 1:
Open up the Properties (the sliders icon on the right), check you are in the Symbol Layer tab, drag to the bottom of the properties and you should see a Time slider. It probably needs to be adjusted to the left. This will insert a keyframe at this frame for this layer.

Controlling the manifestation of the effect version 2:
Alternatively, scrub through the timeline and find if there are any stray keyframes in the Time channel that need to be removed.

Editing the base cause of the effect:
Alternatively, go to the Library tab, tap and hold on the symbol, then tap Edit at the bottom of the window when it appears. Extend the layer length there until it will cover the frame in your main scene where it is not seen.

Note: Often controlling the manifestation is more useful or offers more control than editing the base cause - it's how you get switch layers for lipsync or eye or hand animation (see version 1).
